backend/app/src/main/java/com/moh/phlat/backend/service/SpecificationService.java

```diff
@@ -0,0 +1,22 @@
+package com.moh.phlat.backend.service;
```

Comment:
I think they belong more in repo package. What do you think?

Reply:
I moved to the repo package.  Not really sure where it should be, so I will go with your suggestion.